### PR TITLE
Cypress/E2E: Automate backlogs - System Console > User Mgmt Deactivation (6 test cases)

### PR DESCRIPTION
**Pull request by hanzei**
#### Summary
I did add a couple of helper command to DRY up the code.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-28184

### CODE REVIEW

**Comment by DHaussermann** [comment]
Hi @hanzei the new tests you have added run green and seem to match the corresponding TM4J tasks. But you have excluded the existing tests. Did you intended to flip the `xit.` back to `it.` for these exiting tests?

**Comment by hanzei** [comment]
@DHaussermann Good catch :+1: Enabled them again.

**Comment by DHaussermann** [comment]
Looks good to me. 
I see that MM-T950 was not included as it was archived. @saturninoabril do you know why this test was archived?

**Comment by hanzei** [comment]
@DHaussermann That test was already covered by another test case.

**Review by DHaussermann — APPROVED** [review]
Tested and passed
- Tests match TM4J script and run green
- Updated corresponding TM4J status

LGTM!

File: e2e/cypress/support/ui_commands.js [hunk 1]
```diff
@@ -216,6 +216,106 @@ Cypress.Commands.add('compareLastPostHTMLContentFromFile', (file, timeout = TIME
...
+    cy.get('#post_textbox').
+        type(message).
+        type('{enter}');
+});
+
+/**
+ * Close a DM via the X button
+ * @param {User} sender - TODO
+ * @param {User} recipient - TODO
+ * @param {String} team - TODO
```
Comment (saturninoabril) [inline e2e/cypress/support/ui_commands.js:303]:
```suggestion
 * @param {User} sender - the one currently observing and who will close the DM
 * @param {User} recipient - the other user in a DM 
 * @param {String} team - a team where the sender is member of
```

Comment (saturninoabril) [reply e2e/cypress/support/ui_commands.js:303]: Btw, I'm fine to leave this as is though this command is not actually related to this PR. Clean up can be in a separate PR, specifically with `close_current_dm_redirects_spec.js`

**Review by saturninoabril — APPROVED** [review]
@hanzei Tests look good to me and all passed. I left suggestions which I can apply if you don't mind (given you're in a holiday). Let's deal separately on other comments. Thanks and happy holidays!

**Comment by hanzei** [comment]
Thanks for updating the PR @saturninoabril . The changes your made look good to me :+1:

**Pull request merged**